app/components/ProfilePreview.js

```diff
@@ -14,8 +14,8 @@ const PureProfilePreview = (props) => (
     <hr />
     <h4>Wishlist:
       <ul>{_.get(props, 'profile.wishlist', '') &&
-      _.map(props.profile.wishlist, (list, ind) => (
-        <li key={ind}>{list.value}</li>
+      _.map(props.profile.wishlist, (list) => (
+        <li key={Math.random()}>{list.value}</li>
```

Comment:
Filthy hack that satisfies React's need for a unique `key` on each iterable. Not a problem for arrays, but I need to add a create solution for iterating through objects.